interface/SelectorBase.h

```diff
@@ -229,10 +231,18 @@ public :
     std::string getHistName(std::string histName, std::string variationName);
     template<typename T, typename... Args>
     void SafeHistFill(std::map<std::string, T*> container, 
-            std::string histname, Args... args) {
-        if (container[histname] != nullptr)
-            container[histname]->Fill(args...);
+		      std::string histname, Args... args) {
+      if (container[histname] != nullptr)
+	container[histname]->Fill(args...);
     };
+  
+  template<typename T, typename... Args>
+  void HistFullFill(std::map<std::string, T*> container,
```

Comment:
Is this to have a histogram with the sum of all channels?

Reply:
Yep, I was needing to see things in in terms of all channels so I wrote the function to take care of that. If you want, I can move it to my personal branch. I'm not sure we want to have it for general cases or not

Reply:
I have no strong preference either way. I prefer to keep the information separated by channels because it's can be very useful, and I don't store the sum in addition to the channels because they can be trivially combined when plotting and I don't want to increase the size of the files unnecessarily. So I probably won't use it but it's fine here in any case.